baseapp-chats/baseapp_chats/tests/test_graphql_mutations.py

```diff
@@ -426,3 +432,87 @@ def test_blocked_user_cant_create_room_with_user(django_user_client, graphql_use
         content["data"]["chatRoomCreate"]["errors"][0]["messages"][0]
         == "You can't create a chatroom with those participants"
     )
+
+
+def test_user_can_create_group(django_user_client, graphql_user_client):
+    participant = ProfileFactory()
+    participant_2 = ProfileFactory()
+    with open("./baseapp-chats/baseapp_chats/tests/tree.jpg", "rb") as image:
```

Comment:
instead of using this file you can use `image_djangofile` fixture, just add is there after `graphql_user_client` and you are good to go :)
you can find example of usage in `baseapp-profiles`
then you can delete the `tree.jpg` file

Reply:
@nossila there was a beautiful tree image there, it's a shame ;(